pkg/build/builder/cmd/builder.go

```diff
@@ -39,6 +39,7 @@ func run(builderFactory factoryFunc, scmAuths []scmauth.SCMAuth) {
 		glog.Fatalf("Error obtaining docker client: %v", err)
 	}
 	buildStr := os.Getenv("BUILD")
+	glog.Infof("$BUILD env var is %s \n", buildStr)
```

Comment:
nuke this, or if you think it's printing information we don't already have elsewhere (I thought we do print this somewhere in the s2i build log already..maybe a full dump of the env variables), then at least make it a V(4).